sources/common/inc/EventSignalCreator.h

```diff
@@ -38,12 +38,12 @@ namespace PARSER
 
             /// The event signal creator constructor.
             EventSignalCreator() :
-                SignalCreator("^#([[:d:]]+) ([[:graph:]]+) e")
+                SignalCreator("^#([[:d:]]+) ([[:graph:]]+) e *.*")
```

Comment:
Good point. The intention was: if there's a comment after 'e' it shall be separated by a space. Definitely it won't solve it. I need to come up with a better regex.